docs/plugins.md

```diff
@@ -12,85 +12,106 @@ layout: layouts/extend.njk
 Looking for help using Snowpack in your project?
 👉 **[Check out our main docs.](/)**
 
-## Overview
+## Plugin Overview
 
-There are two types of Snowpack plugins: **build plugins** and **bundler plugins.** Unless you are making a plugin specifically for a bundler like Rollup, Parcel, or webpack, you’re probably making a build plugin.
+A **Snowpack Plugin** is an object interface that lets you customize Snowpack's behavior. Snowpack provides different hooks for your plugin to connect to. For example, you can add a plugin to handle Svelte files, optimize CSS, convert SVGs to React components, run TypeScript during development and much more.
 
-## Build Plugins
+Snowpack's plugin interface is inspired by Rollup. If you've ever written a Rollup plugin before, then hopefully these concepts and terms feel familiar.
 
-Build plugins run on single files that pass through Snowpack. They listen for particular file extensions, and when a match is found, transform that code.
+### Build Plugins
 
-In order to keep operations speedy, Snowpack’s build system is **single pass,** meaning each file is only transformed by a set of plugins once. We’ll talk more about that later.
+Snowpack uses an internal **Build Pipeline** to build files in your application for development and production. Every source file passes through the build pipeline, which means that Snowpack can build more than just JavaScript. Images, CSS, SVGs and more can all be built by Snowpack.
 
-### The Basics
+Snowpack runs each file through the build pipeline in two separate steps:
 
-To create a Snowpack plugin, you may start with the following template:
+1. **Load:** Snowpack finds the first plugin that claims to `resolve` the given file. It then calls that plugin's `load()` method to load the file into your application. This is where compiled languages (TypeScript, Sass, JSX, etc.) are loaded and compiled to something that can run on the web (JS, CSS, etc).
```

Comment:
Not a request for change but just a question sparked by reading this. What about proxying CSS? Is that more of a “load” step or a “transform” step?